### ISSUE TITLE
How to a non-person resource creator as author?

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#1351]
**How to a non-person resource creator as author?**

For the citizen science datasets by [Natuurpunt](`https://www.gbif.org/publisher/4d3ceea8-5699-439d-a899-decac9cbbdac)` we always add `all <taxon group> observers` as the last resource creator, to make sure they are acknowledged in the citation of the dataset. To avoid having to split this into a first and last name, we only populate `Organization` with e.g. `all Hemiptera observers`.

This works fine on the [IPT](`https://ipt.inbo.be/resource?r=hemiptera-natuurpunt-occurrences#citation):`

> Lock K, Chérot F, Libeer R, Claerebout S, Aukema B, Verhoeyen K, Uitterhaegen B, Gielen K, Vanreusel W, Herremans M, Desmet P, Swinnen K, **all Hemiptera observers** (2017): Waarnemingen.be - Hemiptera occurrences in Flanders and the Brussels Capital Region, Belgium. v1.1. Natuurpunt. Dataset/Occurrence. `https://doi.org/10.15468/dkd0sg`

On the GBIF page the observers are also correctly [listed as contributor](`https://www.gbif.org/dataset/37e094f3-dcf2-469f-93a2-c4b9b5fa7275#contributors),` but unlike the IPT, **not listed in the authors of the dataset or the [citation](`https://www.gbif.org/dataset/37e094f3-dcf2-469f-93a2-c4b9b5fa7275#citation)**:`

> Lock K, Chérot F, Libeer R, Claerebout S, Aukema B, Verhoeyen K, Uitterhaegen B, Gielen K, Vanreusel W, Herremans M, Desmet P, **Swinnen K** (2018). Waarnemingen.be - Hemiptera occurrences in Flanders and the Brussels Capital Region, Belgium. Natuurpunt. Occurrence Dataset `https://doi.org/10.15468/dkd0sg` accessed via GBIF.org on 2018-05-17.

---

Can this behaviour be made consistent throughout IPT and GBIF, so non-person creators can be included in the citation, using an empty first/last name and populated organization?

One can argue of course that this type of unnamed contributor should not be added as an author, but I think the benefit of doing so (credit to the contributors) outweigh the disadvantages.

---

Side note: Trying to force this information into first and last name leads to undesirable citations:

- populate in first name only => not allowed
- populate in last name only => empty author
- populate as `all` / `Hemiptera observers` => `Hemiptera observers A`
- populate as organization => works fine, but not on GBIF page (see above)

-----

User provided contact info: @peterdesmet
System: Chrome 66.0.3359 / Mac OS X 10.13.3
User: [See in registry](https://www.gbif.org/api/feedback/user/def8896f9d62c3b666872c15cb0dbb23:8dc87de7f180d300779b5fcf9830f7352993d7376abf320054b1f4e3826546b8fc39d2608da552e923850b47b59e43f0096602f03468f9be160661454462115b)
Referer: https://www.gbif.org/dataset/37e094f3-dcf2-469f-93a2-c4b9b5fa7275
Window size: width 1238 - height 759
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2018-05-17T07:28:19.109Z',mode:absolute,to:'2018-05-17T07:34:19.109Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2018-05-17T07:28:19.109Z',mode:absolute,to:'2018-05-17T07:34:19.109Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: OPERATIONAL

**Comment by MortenHofft** [comment]
@ahahn-gbif and @dschigel I believe you were the principal designers of the citation format - this use case might interest you.

**Comment by dschigel** [comment]
The roles are set for individuals, and if there are any aggregations that act as authors in my opinion would need to be treated as organizations. Think journals - try that with Nature, how this would look like, Join Smith, Jack Daniels, and 27 ginger haired sailors 2018?

**Comment by peterdesmet** [comment]
Ok, but how could a citizen science community then be included? It works for `OpenStreetMap contributors` so I don't see why this would not be allowed for citizen science datasets citations.

**Comment by dschigel** [comment]
@peterdesmet, you are right we have examples of that e.g. iNaturalist.org (2018). iNaturalist Research-grade Observations. Occurrence Dataset https://doi.org/10.15468/ab3s5x accessed via GBIF.org on 2018-05-25. What I meant is that a combination of humans and organizations looks weird and is hard to recommend.
